Core/Graphics/Include/Platform/Android/Babylon/Graphics/Platform.h

```diff
@@ -11,5 +11,7 @@ namespace Babylon::Graphics
         WindowType Window{};
         size_t Width{};
         size_t Height{};
+        // MSAA sample values can be 2, 4, 8 or 16. Any other value will disable MSAA
+        unsigned char MSAASamples{};
```

Comment:
Probably better to use this like in DeviceImpl.h.

```suggestion
        uint8_t MSAASamples{};
```

Reply:
On second thought, maybe this should be an enum? What happens if you pass in something that is not allowed by bgfx?